lib/cmd/cmd.go

```diff
@@ -20,32 +20,27 @@ import (
 	"github.com/spf13/viper"
 )
 
-// Main is the main entry point for the command line tool.
+// Main is the main entry point for the omni application binaries.
 // Usage:
 //
 //	   func main() {
 //		     libcmd.Main(appcmd.New())
 //	   }
 func Main(cmd *cobra.Command) {
-	ctx, cancel := signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGTERM)
-
-	SilenceErrUsage(cmd)
+	wrapRunCmd(cmd)
 
+	ctx, cancel := signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGTERM)
 	err := cmd.ExecuteContext(ctx)
-
 	cancel()
 
 	if err != nil {
-		log.Error(ctx, "!! Fatal error occurred, app died unexpectedly !!", err)
```

Comment:
Also slightly improves fatal errors logs since since we now have "expected fatal errors"